packages/rum-core/src/domain/rumEventsCollection/action/actionCollection.ts

```diff
@@ -95,13 +95,11 @@ function processAction(
       },
       date: action.startClocks.timeStamp,
       type: RumEventType.ACTION as const,
+      view: { in_foreground: pageStateHistory.isInActivePageStateAt(action.startClocks.relative) },
     },
     autoActionProperties
   )
-  const inForeground = foregroundContexts.isInForegroundAt(action.startClocks.relative)
-  if (inForeground !== undefined) {
-    actionEvent.view = { in_foreground: inForeground }
-  }
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about keeping the same logic of not adding the `view.in_foreground` if there is no active page state?

Reply:
The previous `foregroundContexts.isInForegroundAt(action.startClocks.relative)` always return a boolean so `inForeground !== undefined` is always true